ownCloudTests/SettingsTests.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  Settings.swift
+//  ownCloudTests
+//
+//  Created by Jesús Recio on 27/02/2019.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
+import XCTest
+import EarlGrey
+import ownCloudSDK
+import ownCloudMocking
+
+@testable import ownCloud
+
+class SettingsTests: XCTestCase {
+
+	override func setUp() {
+		EarlGrey.select(elementWithMatcher: grey_text("Settings".localized)).perform(grey_tap())
+	}
+
+	/*
+	* PASSED if: Theme and Logging are displayed as part of the "User Interface" section of Settings
+	*/
+	func testCheckUserInferfaceItems () {
+
+		//Assert
+		EarlGrey.select(elementWithMatcher: grey_accessibilityID("theme")).assert(grey_sufficientlyVisible())
+		EarlGrey.select(elementWithMatcher: grey_accessibilityID("logging")).assert(grey_sufficientlyVisible())
+
+		//Reset status
+		EarlGrey.select(elementWithMatcher: grey_text("ownCloud".localized)).perform(grey_tap())
+	}
+
+	/*
+	* PASSED if: "More" options "are displayed
+	*/
+	func testCheckMoreItems () {
+
+		//Assert
+		EarlGrey.select(elementWithMatcher: grey_text("SECURITY".localized))
+			.usingSearch(grey_scrollInDirection(GREYDirection.down, 100), onElementWith: grey_accessibilityID("help".localized))
```

Comment:
@jesmrec will 100 work for all screen sizes?

Reply:
I checked the `amount` parameter with different screen devices (from 5s to 6Plus) and worked as expected.